docs/configure_your_resource.md

```diff
@@ -117,12 +121,13 @@ On your Twig templates, the `books` variable will be replaced by the `library` o
 
 As an example, on an `index` operation these Twig variables will be available:
 
-| Name      | Type                                     |
-|-----------|------------------------------------------|
-| resource  | App\Entity\Book                          |
-| library   | App\Entity\Book                          |
-| operation | Sylius\Component\Resource\Metadata\Index |
-| app       | Symfony\Bridge\Twig\AppVariable          |
+| Name              | Type                                      |
+|-------------------|-------------------------------------------|
+| resource          | App\Entity\Book                           |
+| library           | App\Entity\Book                           |
```

Comment:
Is this right? If plural name is different, but single name still the same? Then it will be book over here and plural name can't be seen.

```suggestion
| book           | App\Entity\Book                           |
```

Reply:
hum there is an issue here, the Twig vars examples are not correct.

Reply:
fixed.